api/controllers/index.js

```diff
@@ -1,3 +1,7 @@
 const userController = require("./userController");
+const mainController = require("./mainController");
```

Comment:
관련 서비스의 이름대로 layer별로 네이밍을 수정해주세요.
```suggestion
const cartController = require("./cartController");
```